.gitattributes

```diff
@@ -0,0 +1,5 @@
+* text=auto eol=lf
```

Comment:
adding a gitattributes file should probably be included in a separate commit rather than bundling in with the rest of the updates.